doc/source/notebooks/svi_test.ipynb

```diff
@@ -302,7 +302,7 @@
     "m.X.minibatch_size = 100\n",
     "m.Y.minibatch_size = 100\n",
     "\n",
-    "m.Z.fixed = True\n",
+    "m.feat.fixed = True\n",
```

Comment:
fixed again

Reply:
This is a problem in many notebooks. Looking at `tests/test_notebooks.py`, @awav is on it.